test/integration/serverless/log_normalize.py

```diff
@@ -0,0 +1,148 @@
+import argparse
+import json
+import re
+import sys
+
+def normalize_metrics(stage):
+    return [
+            replace(r'raise Exception', r'\n'),
+            require(r'BEGINMETRIC.*ENDMETRIC'),
+            exclude(r'BEGINMETRIC'),
+            exclude(r'ENDMETRIC'),
+            replace(r'(ts":)[0-9]{10}', r'\1XXX'),
+            replace(r'(min":)[0-9\.e\-]{1,30}', r'\1XXX'),
+            replace(r'(max":)[0-9\.e\-]{1,30}', r'\1XXX'),
+            replace(r'(cnt":)[0-9\.e\-]{1,30}', r'\1XXX'),
+            replace(r'(avg":)[0-9\.e\-]{1,30}', r'\1XXX'),
+            replace(r'(sum":)[0-9\.e\-]{1,30}', r'\1XXX'),
+            replace(r'(k":\[)[0-9\.e\-]{1,30}', r'\1XXX'),
+            replace(r'(datadog-nodev)[0-9]+\.[0-9]+\.[0-9]+', r'\1X.X.X'),
+            replace(r'(datadog_lambda:v)[0-9]+\.[0-9]+\.[0-9]+', r'\1X.X.X'),
+            replace(r'dd_lambda_layer:datadog-go[0-9.]{1,}', r'dd_lambda_layer:datadog-gox.x.x'),
+            replace(r'(dd_lambda_layer:datadog-python)[0-9_]+\.[0-9]+\.[0-9]+', r'\1X.X.X'),
+            replace(r'(serverless.lambda-extension.integration-test.count)[0-9\.]+', r'\1'),
+            replace(r'(architecture:)(x86_64|arm64)', r'\1XXX'),
+            replace(stage, 'XXXXXX'),
+            exclude(r'[ ]$'),
+    ]
```

Comment:
For the tree different telemetry types, you'll see I've created a list of "normalizers". These normalizers each make a single small change to the logs data that is fed in. It will be easy for us to edit these normalizer lists and can add more complicated normalizers in the future.